pkg/pillar/types/global.go

```diff
@@ -9,6 +9,20 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+// HTTPExtraStatus - Enum name for return extra status from SendOnAllIntf
+type HTTPExtraStatus uint8
```

Comment:
HTTP? Makes it sound like it has something to do with the HTTP protocol.
I think it would make more sense to call this SendResult , and ditto for the enum values, since it is a local interface with the zedcloud.Send* functions.

Reply:
Ok.